link-grammar/api-structures.h

```diff
@@ -70,16 +70,18 @@ struct Parse_Options_s
 {
 	/* General options */
 	short verbosity;       /* Level of detail to give about the computation 0 */
-	char * debug;          /* comma-separated function names to debug "" */
-	char * test;           /* comma-separated features to test "" */
+	char * debug;          /* Comma-separated function names to debug "" */
+	char * test;           /* Comma-separated features to test "" */
 	Resources resources;   /* For deciding when to abort the parsing */
 
 	/* Options governing the tokenizer (sentence-splitter) */
-	short use_spell_guess;  /* Perform spell-guessing of unknown words. */
+	short use_spell_guess; /* Up to this many spell-guesses per unknown word 7 */
 
 	/* Choice of the parser to use */
-	bool use_sat_solver;   /* Use the Boolean SAT based parser */
+	bool use_sat_solver;   /* Use the Boolean SAT based parser FALSE */
+#if USE_VITERBI
 	bool use_viterbi;      /* Use the Viterbi decoder-based parser */
+#endif
```

Comment:
We are approaching a day (if not already past it) when the viterbi code should be removed. It is perhaps a bad idea.

Reply:
I can just remove it from link-grammar-def and ifdef-out the remaining stuff with USE_VITERBY.

Reply:
Yes. I looked at it again, and I just don't see any future for it.